cypress/utils/data_utils.ts

```diff
@@ -212,21 +213,88 @@ export function getRandomCredentialsData(
     }
 }
 
+/**
+ * This function returns JiraConnectionData
+ * @param jiraCredential: credential used to build Jira connection. Can be either CredentialsBasicJira or CredentialsTokenJira
```

Comment:
* @param jiraCredential: credential used to build Jira connection. Can be either CredentialsBasicJira or CredentialsBearerToken